Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/dccwalletinfo/update/DccWalletInfoUpdateTrigger.kt

```diff
@@ -45,7 +46,12 @@ class DccWalletInfoUpdateTrigger @Inject constructor(
                  */
                 .distinctUntilChangedBy { it.sortedQrCodeHashSet }
                 .collectLatest {
-                    runCatching { triggerNow(admissionScenarioId()) }.onFailure {
+                    runCatching {
+                        // delay to collect rapid changes and do only one recalculation
+                        delay(1000)
```

Comment:
I changed to debounce
I added the delay to make sure there won't be consecutive recalculations within short time frames. We always calculate all wallets. It also covers other cases where this might happen. And it is only one line of code.